backend/internal/model/game_test.go

```diff
@@ -59,3 +59,39 @@ func TestAddGoal(t *testing.T) {
 		})
 	}
 }
+
+func Test_game_SubGoal(t *testing.T) {
```

Comment:
It is not a good practice to use a Snake Case for naming the test functions. Instead, use the Camel Case e.g. `TestGameSubGoal`